MaterialLib/MPL/MediumAuxiliaryFunction.h

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
You could name the file also Utils.h to stay consistent with for instance ParameterLib. The ProcessLib has even a subfolder Utils that contains several files.

Reply:
Moved to ProcessLib/Utils